doc-content/enterprise-only/getting-started-kogito/con-kogito-probes.adoc

```diff
@@ -0,0 +1,4 @@
+[id="con-kogito-probes_{context}"]
+= Probes for {KOGITO} microservices on {OPENSHIFT}
+
+The probes in {OPENSHIFT} verify that an application is working or it needs to be restarted. For {KOGITO} microservices on Quarkus and Spring Boot, probes interact with the application using an HTTP request, defaulting to the endpoints that are exposed by an extension. Therefore, to run your {KOGITO} microservices on {OPENSHIFT}, you must import the extensions to enable the https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes[liveness, readiness, and startup probes] for your application.
```

Comment:
`you must import the extensions to enable the https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes[liveness, readiness, and startup probes] for your application`
Probes are enabled for the application by default, the mentioned extension is required to actually provide proper information for probes. So I would adjust the text to something like:
`you must import the extensions to provide application availability information for the https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes[liveness, readiness, and startup probes] in your application`

Reply:
Done